automation/test.sh

```diff
@@ -28,6 +28,16 @@ readonly BAZEL_CACHE="${BAZEL_CACHE:-http://bazel-cache.kubevirt-prow.svc.cluste
 
 
 if [ ${CI} == "true" ]; then
+  git remote add upstream https://github.com/kubevirt/kubevirt.git
+  git fetch upstream
```

Comment:
It would have been better to fetch `main` as part of the original code fetch (I think that is possible).
But this can be examined and improved later.